grafana/resource_organization_preferences_test.go

```diff
@@ -0,0 +1,120 @@
+package grafana
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+	"testing"
+
+	gapi "github.com/grafana/grafana-api-golang-client"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+)
+
+func TestAccResourceOrganizationPreferences(t *testing.T) {
+	CheckOSSTestsEnabled(t)
+	CheckOSSTestsSemver(t, ">=8.0.0")
```

Comment:
Is this `CheckOSSTestsSemver` acceptable? Without it, it seems that `WeekStart` is always `""` in at least Grafana 7.5.15, which results in acceptance test failures like...

```
=== RUN   TestAccResourceOrganizationPreferences
    resource_organization_preferences_test.go:14: Step 1/1 error: Check failed: Check 6/6 error: grafana_organization_preferences.test: Attribute 'week_start' expected "Tuesday", got ""
--- FAIL: TestAccResourceOrganizationPreferences (0.47s)
```

...as seen here: https://drone.grafana.net/grafana/terraform-provider-grafana/1741/11/4